### PR TITLE
Fix style issue in Collector "building" pages 

### LINKED ISSUE #3146: Style issue in Collector "building" pages
For example, see https://opentelemetry.io/docs/collector/trace-receiver/#reading-and-validating-your-receiver-settings:

> image

Notice the lack of space between the code block and the "check your work" callout.

/cc @svrnm

### PR DESCRIPTION
**Pull request by mayankag32**
---

- Fixes #3146 
- Preview: https://deploy-preview-3204--opentelemetry.netlify.app/docs/collector/trace-receiver/#identifying-and-providing-default-settings-for-the-receiver

### CODE REVIEW

**Comment by mayankag32** [comment]
Fix issue - https://github.com/open-telemetry/opentelemetry.io/issues/3146

File: content/en/docs/collector/trace-receiver.md [hunk 1]
```diff
@@ -275,15 +274,14 @@ func (cfg *Config) Validate() error {
...
->   true the Collector will generate an error during its startup process and
->   display the message accordingly.
+{{% alert title="Check your work" color="info" %}}
+- I imported the `fmt` package, so I can properly format print my error messages.
+- I added the `Validate` method to my Config struct where I am checking if the
+ `interval` setting value is at least 1 minute (1m) and if the
+ `number_of_traces` setting value is greater or equal to 1. If that is not
+ true the Collector will generate an error during its startup process and
+ display the message accordingly. {{% /alert %}}
+
```
Comment (chalin) [inline content/en/docs/collector/trace-receiver.md:284]: Overall, this is looking better. Make sure that each `{{% %}}` tag is on its own line. Here is an example, change all other instances too:

```suggestion
{{% alert title="Check your work" color="info" %}}

- I imported the `fmt` package, so I can properly format print my error messages.
- I added the `Validate` method to my Config struct where I am checking if the
 `interval` setting value is at least 1 minute (1m) and if the
 `number_of_traces` setting value is greater or equal to 1. If that is not
 true the Collector will generate an error during its startup process and
 display the message accordingly.
 
 {{% /alert %}}

```

**Review by chalin — CHANGES_REQUESTED** [review]
Overall looking much better. 

The color contrast of the `info` color isn't that great, but that's a separate issue. Maybe try using `color="primary"` (I'm not 100% sure that that's a valid color code, and I can't test it now).

See inline for an example of how to format the alerts.

File: content/en/docs/collector/trace-receiver.md (hunk 1 above)
Comment (mayankag32) [reply content/en/docs/collector/trace-receiver.md]: `color="primary"` is a valid color code, check docsy - https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers

Also, I have formatted the alerts sections as per your suggestions.

**Comment by cartermp** [comment]
@mayankag32 if you run `npm run fix` it should address issues.

**Comment by mayankag32** [comment]
> @mayankag32 if you run `npm run fix` it should address issues.

Thanks for the suggestion!

File: content/en/docs/collector/trace-receiver.md [hunk 2]
```diff
@@ -229,10 +229,12 @@ type Config struct {
...
 ```
 
-> #### Check your work
->
-> - I added the `Interval` and the `NumberOfTraces` fields so I can properly
->   have access to their values from the config.yaml.
+{{% alert title="Check your work" color="primary" %}}
+
+- I added the `Interval` and the `NumberOfTraces` fields so I can properly have
+  access to their values from the config.yaml.
```
Comment (cartermp) [inline content/en/docs/collector/trace-receiver.md:235]:
```suggestion
- Added the `Interval` and the `NumberOfTraces` fields to properly have
  access to their values from the config.yaml.
```

File: content/en/docs/collector/trace-receiver.md [hunk 3]
```diff
@@ -275,15 +277,18 @@ func (cfg *Config) Validate() error {
...
->   messages.
-> - I added the `Validate` method to my Config struct where I am checking if the
->   `interval` setting value is at least 1 minute (1m) and if the
->   `number_of_traces` setting value is greater or equal to 1. If that is not
->   true the Collector will generate an error during its startup process and
->   display the message accordingly.
+{{% alert title="Check your work" color="primary" %}}
+
+- I imported the `fmt` package, so I can properly format print my error
+  messages.
```
Comment (cartermp) [inline content/en/docs/collector/trace-receiver.md:283]:
```suggestion
- Imported the `fmt` package to properly format print error messages.
```

File: content/en/docs/collector/trace-receiver.md [hunk 4]
```diff
@@ -275,15 +277,18 @@ func (cfg *Config) Validate() error {
...
+
+- I added the `Validate` method to my Config struct where I am checking if the
+  `interval` setting value is at least 1 minute (1m) and if the
+  `number_of_traces` setting value is greater or equal to 1. If that is not true
+  the Collector will generate an error during its startup process and display
+  the message accordingly.
```
Comment (cartermp) [inline content/en/docs/collector/trace-receiver.md:289]:
```suggestion
- Added the `Validate` method to the Config struct to check if the
  `interval` setting value is at least 1 minute (1m) and if the
  `number_of_traces` setting value is greater or equal to 1. If that is not true
  the Collector will generate an error during its startup process and display
  the message accordingly.
```

**Review by cartermp — CHANGES_REQUESTED** [review]
Some brief copy-edits, but otherwise this looks good.

**Review by cartermp — APPROVED** [review]

File: content/en/docs/collector/trace-receiver.md [hunk 5]
```diff
@@ -275,15 +277,17 @@ func (cfg *Config) Validate() error {
...
-> - I added the `Validate` method to my Config struct where I am checking if the
->   `interval` setting value is at least 1 minute (1m) and if the
->   `number_of_traces` setting value is greater or equal to 1. If that is not
->   true the Collector will generate an error during its startup process and
->   display the message accordingly.
+{{% alert title="Check your work" color="primary" %}}
+
+- Imported the `fmt` package to properly format print error messages.
+
+- Added the `Validate` method to the Config struct to check if the `interval`
```
Comment (chalin) [inline content/en/docs/collector/trace-receiver.md:284]: You've added blank lines between list entries, this spaces them out also in the generated pages, we don't need that extra spacing, so please remove the extra blank lines you added, here and elsewhere:

```suggestion
- Imported the `fmt` package to properly format print error messages.
- Added the `Validate` method to the Config struct to check if the `interval`
```

**Review by chalin — COMMENTED** [review]
Getting closer! See inline comment.

**Review by chalin — APPROVED** [review]
Very nice! 

Didn't re-read the full content of each "Check your work" section, but from a point of view of the styling changes, this LGTM!

**Pull request merged**